### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yehancha**
A little description about the firstRepo is amended.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 firstRepo
```
Comment (praminda) [inline README.md]: First PR comment to the first repo

File: README.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
...
 =========
 
-My first test repo
```
Comment (praminda) [inline README.md:4]: First PR comment to the first repo

**Comment by yehancha** [comment]
Ha ha, did I wrote that commit message... :D

**Comment by praminda** [comment]
There are conflicts in the PR. Please resolve and send a new PR :smile:

**Pull request closed without merging**